docs/contracts.rst

```diff
@@ -879,6 +879,13 @@ the order from "most base-like" to "most derived".
 
 .. index:: ! contract;abstract, ! abstract contract
 
+Inheriting Different Kinds of Members of the Same Name
+======================================================
+
+When an inheritance chain contains a function and a modifier of the same name, it is considered as an error.
```

Comment:
What do you mean by "inheritance chain" here? Can this be rephrased as something like "when inheritance would result in a contract having..."?